.gitignore

```diff
@@ -224,3 +224,4 @@ node_modules/color-support/
 node_modules/fs.realpath/
 node_modules/jasmine-core/
 node_modules/jasmine/
+CordovaLib/CordovaLib.xcodeproj/project.xcworkspace
```

Comment:
Given that CordovaLib has its own .gitignore, let's just add this in there—there aren't any other CordovaLib entries in this file.